lib/routes/receive_payment/widgets/address_widget/address_qr.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:l_breez/routes/receive_payment/lightning/widgets/compact_qr_image.dart';
+
+class AddressQR extends StatelessWidget {
+  final String bolt11;
```

Comment:
Maybe rename this to `destination` or `data`

Reply:
I agree, these widgets will be renamed & refactored next.

Reply:
Done. [Refactor AddressWidget](https://github.com/breez/misty-breez/pull/167/commits/ea1ef395a50d2e950f0615a8653dbfb07a807abd)